import-export-cli/impl/mg/listAPIs.go

```diff
@@ -0,0 +1,129 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"text/template"
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/formatter"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+const (
+	apiNameHeader    = "NAME"
+	apiVersionHeader = "VERSION"
+	apiTypeHeader    = "TYPE"
+	apiLabelsHeader  = "LABELS"
+
+	defaultAPITableFormat = "table {{.Name}}\t{{.Version}}\t{{.Type}}\t{{.Labels}}"
+)
+
+type APIMeta struct {
+	Total int32             `json:"total"`
```

Comment:
I have noticed in several places there are variables defined as **int32** type. Is there a specific reason for that or shall we change those occurrences to just **int**?
The type **int** is also by default 32 bits wide [1]. When you need an integer value you should use **int** unless you have a specific reason to use a sized integer type.

[1] https://tour.golang.org/basics/11

Reply:
I referred to the object [here](https://github.com/wso2/product-apim-tooling/blob/0625bac585c5a8c50ec343b6830f25ed0afa9b82/import-export-cli/utils/structs.go#L104). If that's the case I'll update it to int. Thanks.